packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -104,6 +105,25 @@ if (!ExecutionEnvironment.canUseDOM) {
     localClearTimeout(timeoutId);
   };
 } else {
+  if (__DEV__) {
+    if (typeof requestAnimationFrameForReact !== 'function') {
+      warning(
+        false,
+        'React depends on requestAnimationFrame. Make sure that you load a ' +
+          'polyfill in older browsers. https://fb.me/react-polyfills',
+      );
+    }
+  }
+
+  let localRequestAnimationFrame =
+    typeof requestAnimationFrameForReact === 'function'
+      ? requestAnimationFrameForReact
+      : function(callback: Function) {
+          throw new Error(
```

Comment:
Please use `invariant(false, message)` here so the error message gets stripped out in prod.